### ISSUE TITLE
Rename to DecisionTrees.jl?

### ISSUE THREAD

**Issue by harryscholes** [bensadeghi/DecisionTree.jl#115]
As per the [package naming guidelines](https://julialang.github.io/Pkg.jl/v1/creating-packages/#Package-naming-guidelines-1) in the Pkg.jl docs.

**Comment by KronosTheLate** [comment]
This would be non-breaking, as the old code would still be able to use the old package.
Any arguments against renaming?

**Comment by ablaom** [comment]
Only that DecisionTree.jl is all over the show in MLJ src code and documentation, for example.

**Comment by KronosTheLate** [comment]
So someone rolling up their sleeves, running search and replace throughout the documentation and docstrings (all files really), is the only thing preventing a renaming?

**Comment by ablaom** [comment]
Yep. But I don't think it's that simple.  For continuity I'd be looking for a fairly rapid roll out the effected repos and web sites. That's at least a dozen or more repos/websites for MLJ, I'm guessing. So, if someone wants to line up all those PR's... For me personally, there are [plenty](https://github.com/orgs/JuliaAI/projects/1) of higher priorities.
